### DISCUSSION TITLE
Has anyone tried hosting Pocketbase on a ASUSTOR NAS? Or NAS

### DISCUSSION THREAD

**Discussion by notjj1234** [pocketbase/pocketbase#5645] [Q&A] [answered]
So from following guides on youtube i create a little social media instgram facebook clone made with TypeScript. the backend used was appwrite but i dont wanna pay money for additional storage. 

Im thinking of using Pocketbase and hosting it on my ASUSTOR NAS, though i dont know how to do that and how do i make it public so people can use my clone outside my local network

Im a big noob when it comes to this, so any help would be very cool

**Comment by thangngoc89** [comment, marked as answer]
I wouldn’t recommend hosting pocketbase directly on the NAS because they are usually using a heavily customized Linux distribution. A cheap VPS would suffice to host the main Pocketbase instant. You may do a NFS mount from the NAS to store the uploaded files but the SQLite databases should be on a directly attached NVMe to the machine.
